webapp/components/wpt-amend-metadata.js

```diff
@@ -84,6 +85,11 @@ class AmendMetadata extends LoadingState(ProductInfo(PolymerElement)) {
         type: Array,
         value: []
       },
+      // This testStatusValues mapping is defined at shared/statuses.go.
+      testStatusValues: {
+        type: Object,
+        value: { 'FAIL': 6, 'TIMEOUT': 4, 'ERROR': 3 }
```

Comment:
The triage API expects a numerical value for test status? These numbers are supposed to be internal.

Reply:
Per our 1:1, the `status` field is optional so I will investigate the original motives of using numeral values vs strings and add it in a separate PR.